handbook/business-operations/README.md

```diff
@@ -828,6 +828,11 @@ When the final signature is added to an envelope in DocuSign, it is marked as co
 
 Domain name registrations are handled through Namecheap. Access is managed via 1Password.
 
+#### Vetty
+
+Fleet employees with access to Fleet's infrastructure undergo background check provided through Vetty.  
+Only the most recent background checks appear on the home page of Vetty's dashboard. To access a complete list of background checks run in Vetty scroll down to the bottom of the candidates page and click "View Historical."
```

Comment:
Combine both of these onto the same line, since there's no extra newline between them anyways (meaning they'll be rendered on a single line)

Reply:
add a comma:

```
run in Vetty, scroll down to the bottom of 
```

Reply:
Confirmed with our resident English major that in this situation, put the period outside the double quotes at the end.